pwiz_tools/Skyline/SkylineTester/SkylineTesterWindow.Designer.cs

```diff
@@ -871,7 +871,8 @@ private void InitializeComponent()
             this.tutorialsDemoMode.Name = "tutorialsDemoMode";
             this.tutorialsDemoMode.Size = new System.Drawing.Size(82, 17);
             this.tutorialsDemoMode.TabIndex = 6;
-            this.tutorialsDemoMode.Text = "Demo mode";
+            this.tutorialsDemoMode.Text = "Demo mode (deprecated, use Tests tab)";

```

Comment:
Let's just "Demo mode (deprecated)". Presumably the curious can get the other information from the tip. I know this is not user-facing, but the full text for me comes close to the line where I am tempted to consider a better looking solution.